src/Compilers/CSharp/Test/Emit/PDB/PDBIteratorTests.cs

```diff
@@ -233,7 +242,16 @@ System.Collections.Generic.IEnumerable<int> IEI<T>(int i0, int i1)
     }
 }
 ";
-            var c = CreateCompilationWithMscorlibAndSystemCore(text, options: TestOptions.DebugDll);
+
+            var c = CompileAndVerify(text, options: TestOptions.DebugDll, symbolValidator: module =>
+            {
+                Assert.Equal(new[]
+                {
+                    "<>3__i0",
+                    "<>3__i1",
```

Comment:
This is odd. ```x``` and ```y``` are not lifted in debug builds?

Reply:
The PDB has 2 hoisted local slots.

---
In reply to: [150689364](https://github.com/dotnet/roslyn/pull/22985#discussion_r150689364) [](ancestors = 150689364)

Reply:
Should be lifted. This is "Debug" though. The change does not affect "Debug". The parameters are lifted.
Are Enumerable and Enumerator separate in debug by any chance?